test/lib/testMayaUsdCreateStageCommands.py

```diff
@@ -72,3 +77,14 @@ def testCreateStageFromFile(self):
 
         # Verify that the shape node is connected to time.
         self.assertTrue(cmds.isConnected('time1.outTime', shapeNode+'.time'))
+
+        # Now switch to having USD stage paths as relative to Maya scene file
+        cmds.optionVar(iv=('mayaUsd_MakePathRelativeToSceneFile', 1))
+        
+        # Create the same stage and verify that now it's open as relative
+        shapeNodeRel = mel.eval('mayaUsd_createStageFromFilePath(\"'+ballFilePath+'\")')
+        filePathAttrRel = cmds.getAttr(shapeNodeRel+'.filePath')
+        self.assertEqual(filePathAttrRel, 'top_layer.usda')
+        
+        # Restore mayaUsd_MakePathRelativeToSceneFile
+        cmds.optionVar(iv=('mayaUsd_MakePathRelativeToSceneFile', 0))
```

Comment:
FYI, you shouldn't have to do this because every test will run with its own set of newly created prefs.

Reply:
Couple times in the past I stumbled upon problems with prefs that I thought would be reset to default but they werent't. Things like default background color and display layer color. Prefs set during one run of the test would persist at the next run. So now I prefer to be sure